node_modules/oae-core/filepreview/css/filepreview.css

```diff
@@ -90,4 +90,11 @@
 
 .vjs-tech {
     background-color: #000;
+    width: 98% !important;
+    left: 20px !important;
+}
+
+.video-js .vjs-control-bar {
```

Comment:
This doesn't seem like the best approach to make the widgets align. The best approach would be to add padding on the left and right of the video player to make it align with the comments widget.